### PR TITLE
fix: fix login in docker

### PR DESCRIPTION
**Pull request by ccgomezn**
https://github.com/user-attachments/assets/0d479aea-5e01-4741-8011-00a7679637fb

### CODE REVIEW

File: frontend/src/app/api/auth/[...nextauth]/route.ts [hunk 1]
```diff
@@ -35,11 +35,15 @@ const handler = NextAuth({
       },
       async authorize(credentials, req) {
         try {
+          console.log("Authorizing credentials:", credentials);
```
Comment (ianrtracey) [inline frontend/src/app/api/auth/[...nextauth]/route.ts:38]: can we remove these console logs? we dont want to accidentally log user credentials

File: frontend/src/lib/auth.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@ import wretch from "wretch";
 import { setCookie, deleteCookie, getCookie } from 'cookies-next';
 
 // Base API setup for making HTTP requests
-const api = wretch(process.env.NODE_ENV === "development" ? "http://localhost:8000" : "https://turntable-django-4426.onrender.com").accept("application/json");
+const api = wretch(process.env.NODE_ENV === "development" ? process.env.API_URL ?  process.env.API_URL : "http://localhost:8000" : "https://turntable-django-4426.onrender.com").accept("application/json");
```
Comment (ianrtracey) [inline frontend/src/lib/auth.ts:6]: Does this ternary still work as expected?

**Review by ianrtracey — CHANGES_REQUESTED** [review]

File: frontend/src/app/api/auth/[...nextauth]/route.ts (hunk 1 above)
Comment (ccgomezn) [reply frontend/src/app/api/auth/[...nextauth]/route.ts:38]: oh yes you are right, sorry

File: frontend/src/lib/auth.ts (hunk 2 above)
Comment (ccgomezn) [reply frontend/src/lib/auth.ts:6]: yes but I will add parenthesis for clarity

**Pull request merged**